winml/adapter/winml_adapter_apis.h

```diff
@@ -16,7 +16,7 @@ ORT_API(void, ReleaseExecutionProvider, OrtExecutionProvider*);
 ORT_API_STATUS(OverrideSchema);
 
 // OrtEnv methods
-ORT_API_STATUS(EnvConfigureCustomLoggerAndProfiler, _In_ OrtEnv* env, OrtLoggingFunction logging_function, OrtProfilingFunction profiling_function, _In_opt_ void* logger_param, OrtLoggingLevel default_warning_level, _In_ const char* logid, _Outptr_ OrtEnv** out);
+ORT_API_STATUS(EnvConfigureCustomLoggerAndProfiler, _In_ OrtEnv* env, OrtLoggingFunction logging_function, OrtProfilingFunction profiling_function, _In_opt_ void* logger_param, OrtLoggingLevel default_warning_level, _In_ const char* logid);
```

Comment:
Is it an ABI breaking change?

Reply:
Good point, I'll change it back. I originally removed the param because it wasn't used at all.